controllers/users.js

```diff
@@ -27,7 +27,7 @@ const verifyUser = async (req, res) => {
     if (!req.userData?.profileURL) {
       return res.boom.serverUnavailable("ProfileURL is Missing");
     }
-    await userQuery.addOrUpdate({ profileStatus: "PENDING" }, userId);
+    await userQuery.addOrUpdate({ profileStatus: "PENDING", roles: { archived: false, in_discord: true } }, userId);
```

Comment:
This is a profile service check function?

Reply:
No, this is verify function in /users